libraries/AP_SerialManager/AP_SerialManager.cpp

```diff
@@ -27,11 +27,11 @@
 extern const AP_HAL::HAL& hal;
 
 #ifdef CONFIG_ARCH_BOARD_PX4FMU_V4
-#define SERIAL4_PROTOCOL SerialProtocol_MAVLink
-#define SERIAL4_BAUD 921600
+#define SERIAL5_PROTOCOL Serialprotocol_Mavlink
```

Comment:
Protocol needs upper case. Also doesn't the defaults in HAL_PX4_Class.cpp need to changed?